README.md

```diff
@@ -41,6 +41,25 @@ The following bounties are available for anyone to pick up:
 
 <br>
 
+# Bounty Submission
+
+## Outside of [XRPLF](https://github.com/XRPLF)
+
+To submit your bounty work(finished or work in progress) for review, you will share your submission by opening a PR to new open source repo(s) under the [XRPL Bounties github organization](https://github.com/XRPLBounties).
+
+This will make it easier for reviewers and submitters to collaborate (comment on specific line of codes and request changes). It will also increase the publicity of your work, as the community will be able to find everything under XRPL Bounties.
+
+### Instructions
+Reply to the corresponding discussion thread with a link to your work-in-progress repo. If you've made enough progress for us to start requesting revisions, we will create an open source repo (under the XRPLBounties github org) for you to open PR(s) against.
```

Comment:
We should create the repo immediately for projects which will require a separate repository of work. (Ex. PoA bounty)

Reply:
(If we commit to doing this, we can get rid of this Instructions section - or just have it be a paragraph saying something like "Submit by opening a PR against the corresponding Repo" (I don't know exact wording)